css/src/atomics/spacing.scss

```diff
@@ -34,8 +34,30 @@ $layout-sizes: (
 	.#{$paddingKey}#{$separator}#{$sizeKey} {
 		padding: sizeValue($sizeKey, $sizeValue) !important;
 	}
+}
 
-	// margin-inline and margin-block property isn't supported by IE, Edge, and Safari yet (as of Apr 2021)
+@each $position-key, $direction in $directions {
```

Comment:
Moved this out of the first loop to ensure `margin-<side>-<size>` is more specific than `margin-<size>`.